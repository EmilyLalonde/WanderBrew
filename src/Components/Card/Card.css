.card {
  border: 1px solid black;
  display: grid;
  margin-left: 10%;
  text-align: center;
  text-decoration: none;
  width: 80%;
}


a {
  text-decoration: none;
 }