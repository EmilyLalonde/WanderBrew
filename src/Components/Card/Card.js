import React from 'react'
import './Card.css'
import favorite from '../../images/favoriteIcon.svg';
import isNotFavorite from '../../images/isNotFavorite.svg';

export const Card = ({id, name, street, city, state, phone}) => {
  return (
    <div className="card">
      <h1>{name}</h1>
      <p>{street}</p>
      <p>{city}</p>
      <p>{state}</p>
      <p>{phone}</p>
    </div>
  )
}

export default Card