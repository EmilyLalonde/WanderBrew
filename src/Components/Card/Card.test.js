import React from 'react'
import { shallow } from 'enzyme'
import { Card } from './Card'

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card 
      id= {1516}
      name= 'Ratio Beerworks'
      street= '2920 Larimer St'
      city= 'Denver'
      state='Colorado'
      phone='3012669264'
      />)
  });
  it('should match the snapshot with all data being passed correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})