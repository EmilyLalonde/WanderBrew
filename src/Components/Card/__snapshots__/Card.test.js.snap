// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot with all data being passed correctly 1`] = `
<div
  className="card"
>
  <h2 />
  <p
    className="card-p"
  />
  <p
    className="card-p"
  />
  <p
    className="card-p"
  />
  <p
    className="card-p"
  />
</div>
`;
